src/mlpack/tests/main_tests/perceptron_test.cpp

```diff
@@ -40,6 +40,14 @@ struct PerceptronTestFixture
   }
 };
 
+// reset the parameters
+void resetSettings()
+{
+  bindings::tests::CleanMemory();
+  CLI::ClearSettings();
+  CLI::RestoreSettings(testName);
+}
```

Comment:
It looks like this function is never used; should we remove it?

Reply:
Yeah, you are right. It is never used. I overlooked that.